_commands/editors/vi.md

```diff
@@ -0,0 +1,46 @@
+vi
+
+
+
+'vi' is a text editor for programmers, designed to facilitate optimal efficiency when writing code. 
+
+Since its creation, the developers have also released "vim" (vi improved), an updated version with more features, such as support for more programming languages, compatibility with more OS's, and features like multilevel undo/redo. On some Linux distributions, the vi executable provided is actually Vim. 
+
+Despite this, some programmers still prefer vi over vim for its simplicity and ease of access.
+
+To open vi, simply type the following:
+
+~~~ bash
+$ vi [filename]
+~~~
+
+This will open a file with the name specified. If there is an existing file with that name, it will open that file, however if no file is found with that name, a new, empty one will be created.
+
+
+<!--more-->
+
```

Comment:
You should probably move the first few paragraphs below the `<!---more-->`. That way the preview of the reference entry is just ...
![screen shot 2016-03-27 at 6 14 21 pm](https://cloud.githubusercontent.com/assets/7095187/14068133/c7120878-f447-11e5-9bd9-39de7f074e02.png)